### DISCUSSION TITLE
Java Flutter Web Multi Project maven compiler plugin

### DISCUSSION THREAD

**Discussion by ZooMMaX** [community/community#121586] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello people!  I have written a plugin for Maven, the purpose of which is to pack multilingual (java + flutter) client-server solutions into one jar file.  I find this approach practical because...  it speeds up the deployment process.  I would really like to receive feedback, criticism and suggestions, if this is interesting to at least someone other than me =)

[jfwmp-maven-plugin](https://github.com/ZooMMaX/jfwmp-maven-plugin)

**Comment by Dilan797** [comment]
Response #1
